lighthouse-core/test/fraggle-rock/scenarios/__snapshots__/stringified-replay-test-pptr.js.snap

```diff
@@ -0,0 +1,485 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Running the stringified output script generates a valid desktop flow report 1`] = `
```

Comment:
seems like a lot of this is snapshotting `@puppeteer/replay` code

Reply:
Yeaaaaah...

The unit test file does some trimming to get rid of a lot of the boilerplait stuff. However, I wanted some tests to validate the *entire* output though, just so we know what @puppeteer/replay changes when we bump versions.